tests/scripts/multi-node/build-rook.sh

```diff
@@ -87,6 +87,10 @@ function make_rook {
 function run_rook {
   cd "$rook_kube_templates_dir"
   kubectl create -f rook-operator.yaml
+  while ! kubectl get crd clusters.rook.io >/dev/null 2>&1; do
```

Comment:
Do we really want to wait forever here? I believe we should timeout the thing after 30sec or so, don't you think?

Reply:
we could.  It's more complex, and ^C works, and it's echoing, so it's clear what's going on.  Depends on how often you think this will fail after the preceding create succeeds, perhaps; I assumed that would be unlikely.

Reply:
I personally never experienced this issue so I don't really know but I guess it's fine with the 'echo'.